Cargo.toml

```diff
@@ -24,6 +24,7 @@ include = [
     "!tests/.pytest_cache",
     "!*.so",
 ]
+rust-version = "1.70"
```

Comment:
A minimum version, and at the moment all this will do is cause `Cargo` to fail the build if the installed rust version is lower. 

Future Rust versions might do more here (e.g. fail the build if we accidentally use stuff newer than this version.)